index.html

```diff
@@ -288,34 +288,16 @@
     </p>
 
     <h2>Installing</h2>
-    <p>
-    Assuming you have the prerequisites and are running <code>bash</code>:
-    </p>
-    <pre class="code">
-curl -O https://sw.lsstcorp.org/eupspkg/newinstall.sh
-bash newinstall.sh
-
-source loadLSST.bash
-
-eups distrib install -t v12_0 lsst_apps</pre>
-    <p>
-    This will download and build from source a specific release of the LSST
-    Stack (v12.0, in the example above).
-    </p>
 
-    <p>
-    For complete instructions, see the <a href="https://pipelines.lsst.io/">documentation</a>.
-    </p>
+    <p>There are several ways of installing the LSST Stack, including from source, through Anaconda, or as Docker containers. Our <a href="https://pipelines.lsst.io/install/index.html">installation documentation</a> will get you started.</p>
```

Comment:
I understand the intent, though I felt a little unsure about whether it was a entirely honest. If we *do* want to have a one-line install doc on the homepage, should we switch to conda as the 'default' installation?